rebasehelper/specfile.py

```diff
@@ -331,7 +331,7 @@ def get_not_used_patches(self):
         """
         return self.patches['not_applied']
 
-    def _comment_out_patches(self, patch_num, note=None):
+    def _comment_out_patches(self, patch_num, note=None, remove_patches=False):
```

Comment:
Well, the method is called `_comment_out_patches` and with `remove_patches=False` it does exactly that.